src/Utilities/LocalizationLoader.ts

```diff
@@ -0,0 +1,20 @@
+type localizationObjectType = {
+  [key: string]: {
+    [key: string]: string
+  }
+}
+
+const initLocalizationObject = (): localizationObjectType => {
+  const localizations = require.context('../Localization', false, /\.json$/);
+
+  const localizationObject:{[index: string]: any} = {};
```

Comment:
I quess we can use [index: string]: string right?

Reply:
[9ede5d4](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/9ede5d4e8127366824a28d497840cf0cf5e1cc2e)